editor.planx.uk/src/pages/Pay/InviteToPay.tsx

```diff
@@ -60,19 +62,18 @@ const InviteToPay: React.FC<PaymentRequest> = ({ createdAt }) => {
             if you have any questions about your application or this service
           </li>
         </List>
-        {/* TODO: Get this from team data (store?) */}
         <Box pt={2}>
           <Typography variant="body2" sx={{ margin: 0 }}>
-            <b>Telephone</b> 0300 131 6000
+            <b>Telephone</b> {team.notifyPersonalisation?.helpPhone}
```

Comment:
Should all of these be in a conditional block if `team.notifyPersonalisation` isn't set?